solutions/monorepo/README.md

```diff
@@ -5,9 +5,9 @@ This is a monorepo example with a single Next.js site ([./app](./app)) that has
 - [./packages/ui](./packages/ui): Exports UI components that use TypeScript and Tailwind CSS
 - [./packages/utils](./packages/utils): Exports utilty functions that use TypeScript
 
-The monorepo is using [Turborepo](https://turborepo.org/) and [yarn workspaces](https://classic.yarnpkg.com/lang/en/docs/workspaces/) to link packages together, but it can also work with [npm workspaces](https://docs.npmjs.com/cli/v7/using-npm/workspaces#using-workspaces).
+The monorepo is using [Turborepo](https://turborepo.org/) and [npm workspaces](https://docs.npmjs.com/cli/v7/using-npm/workspaces#using-workspaces) to link packages together, but it can also work with [yarn workspaces](https://classic.yarnpkg.com/lang/en/docs/workspaces/).
```

Comment:
```suggestion
The monorepo is using [Turborepo](https://turborepo.org/) and [npm workspaces](https://docs.npmjs.com/cli/8/using-npm/workspaces#using-workspaces) to link packages together, but it can also work with [yarn workspaces](https://classic.yarnpkg.com/lang/en/docs/workspaces/).
```

Reply:
to match the version at `packageManager` in package.json

Reply:
Thank you!